src/utils/useTimelapseCounter.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useEffect, useState } from 'react';
+
+const useTimelapseCounter = initialTimestamp => {
+  const [renderCount, setRenderCount] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const secondsAgo = Math.floor((Date.now() - initialTimestamp) / 1000);
+      setRenderCount(secondsAgo);
+    }, 1000);
```

Comment:
suggestion(minor): change the timer to 200ms.

This is just to avoid a timing edge case where we miss the timing and the counter takes 2 seconds to update.
If we use 200ms the highest error is 1.2s which is less jarring.

Reply:
Right, timer value changed to 200ms. Please review.🙂